lerna.json

```diff
@@ -1,6 +1,7 @@
 {
   "version": "3.12.0-dev",
   "packages": [
-    "packages/*"
+    "packages/*",
+    "e2e/scripts"
```

Comment:
why do we need this?

Reply:
So when we run npm run lint or npm test from root, lerna needs to identify the e2e dir as a package too so it can call linting commands and tests from that dir. This is where we define e2e is also technically a package now :D